gatsby-node.js

```diff
@@ -284,6 +284,18 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
     redirectInBrowser: true,
     isPermanent: true,
   });
+  createRedirect({
+    fromPath: "/kanvas",
+    toPath: "/cloud-native-management/meshmap",
+    redirectInBrowser: true,
+    isPermanent: true,
+  });
+  createRedirect({
+    fromPath: "/cloud-native-management/meshmap",
+    toPath: "/cloud-native-management/kanvas",
+    redirectInBrowser: true,
+    isPermanent: true,
+  });
```

Comment:
```suggestion
  createRedirect({
    fromPath: "/kanvas",
    toPath: "/cloud-native-management/meshmap",
    redirectInBrowser: true,
    isPermanent: true,
  });
  createRedirect({
    fromPath: "/cloud-native-management/kanvas",
    toPath: "/cloud-native-management/meshmap",
    redirectInBrowser: true,
    isPermanent: true,
  });
```

Reply:
we are looping from 
`/kanvas` to `/cloud-native-management/meshmap` to `/cloud-native-management/meshmap` to `/cloud-native-management/kanvas`